agent/stats/task_windows_test.go

```diff
@@ -108,14 +85,16 @@ func TestTaskStatsCollection(t *testing.T) {
 			TaskMetadata: &TaskMetadata{
 				TaskArn:          taskId,
 				ContainerPID:     containerPID,
-				DeviceName:       []string{"Ethernet 3"},
+				DeviceName:       []string{deviceName},
```

Comment:
Why are we using DeviceName? Isnt the ACS api based on mac and then we find index? Once we find Index we can get LUID. And from there we are good to go for basically all win32 api's. Why use name?

Reply:
Well that is the current workflow which is shared with the Linux. We store the stats corresponding to each device name. Reference: https://github.com/aws/amazon-ecs-agent/blob/67240c17711c0d0a5c7b1d37882dc565880bff38/agent/stats/task.go#L151

Regarding fetching interface index, I have mentioned the same in earlier comment.